wiki/Beatmap/Hype/ru.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - train
+---
+
+# Хайп
+
+Каждая [карта](/wiki/Beatmap) на [рассмотрении](/wiki/Beatmap/Category#work-in-progress-и-pending) имеет свой раздел для [обсуждения](/wiki/Beatmap_discussion), где пользователи могут публиковать специальные посты на вкладке `Общее (все сложности)` с помощью функции `Хайпануть!`. Это приблизительно отражает количество людей, желающих увидеть эту карту [ранкнутой](/wiki/Beatmap/Category#ranked), и помогает продвинуть её на следующий этап процесса ранкинга.
```

Comment:
eventually we'll have to translate all categories' names but hopefully not today
```suggestion
Каждая [карта](/wiki/Beatmap) в категории [Pending](/wiki/Beatmap/Category#work-in-progress-и-pending) имеет свой раздел для [обсуждения](/wiki/Beatmap_discussion), где пользователи могут публиковать специальные посты на вкладке `Общее (все сложности)` с помощью функции `Хайпануть!`. Это приблизительно отражает количество людей, желающих увидеть эту карту [ранкнутой](/wiki/Beatmap/Category#ranked), и помогает продвинуть её на следующий этап процесса ранкинга.
```